tests/functional/conftest.py

```diff
@@ -413,6 +388,10 @@ def _make_merge_request(*, source_branch: str, create_pipeline: bool = False):
                 "commit_message": "New commit in new branch",
             }
         )
+
+        # Helps with Debugging why MRs fail to merge resulting in 405 from downstream tests
+        project.approvalrules.list()
```

Comment:
This doesn't actually print anything out though, right? Feels a bit wrong to just leave it here, unless I missed something. I think we can remove it if it was used to debug during this PR. or do we want to assert here that we really have approval rules?